src/add/observable/combineLatest.ts

```diff
@@ -1,6 +1,4 @@
 import {Observable} from '../../Observable';
 import {combineLatestStatic} from '../../operator/combineLatest';
 
-Observable.combineLatest = combineLatestStatic;
-
-export var _void: void;
```

Comment:
That's cool news, but it's starting from tomorrow's nightly version, right? thinking about discussion for min version support at https://github.com/ReactiveX/RxJS/pull/1288#issuecomment-178859936, that might need to be hold off for moment :(